nodeletctl/pkg/nodeletctl/nodeletctl.go

```diff
@@ -391,7 +391,15 @@ func DeployCluster(clusterCfg *BootstrapConfig) error {
 		nodeletCfg.NodeletRole = "master"
 		nodeletCfg.MasterList = &masterList
 		nodeletCfg.EtcdClusterState = "new"
-
+		for _, path := range nodeletCfg.UserImages {
```

Comment:
We have a similar systemImages field here. Can you test the images there too?

(Possibly that's only in the v1.0 version)

Reply:
done.See pull #99